src/registrar/templates/django/forms/label.html

```diff
@@ -2,15 +2,25 @@
   class="{% if label_classes %} {{ label_classes }}{% endif %}{% if label_tag == 'legend' %} {{ legend_classes }}{% endif %}"
   {% if not field.use_fieldset %}for="{{ widget.attrs.id }}"{% endif %}
 >
-  {% if span_for_text %}
-    <span>{{ field.label }}</span>
+  {% if legend_heading %}
```

Comment:
Integrates legends into our labels so that their contents are tied to screenreader output